engines/sci/detection.cpp

```diff
@@ -110,7 +110,7 @@ static const PlainGameDescriptor s_sciGameTitles[] = {
 	{"msastrochicken",  "Ms. Astro Chicken"},
 	{"pq1sci",          "Police Quest: In Pursuit of the Death Angel"},	// Note: There was also an AGI version of this
 	{"pq3",             "Police Quest III: The Kindred"},
-	{"sq1sci",          "Space Quest I: The Sarien Encounter"},	// Note: There was also an AGI version of this
+	{"sq1sci",          "Space Quest: Chapter I - The Sarien Encounter"},	// Note: There was also an AGI version of this
```

Comment:
The SCI version has the title "Space Quest I: Roger Wilco in the Sarien Encounter" according to its box art (see https://www.mobygames.com/game/space-quest-i-roger-wilco-in-the-sarien-encounter/cover-art/gameCoverId,934/)

Reply:
So question: Is mobygames.com a good site to do my research? Admittedly I looked on a couple of wikis and tried to decide from there(clearly not sufficient for what I was trying to do).

Reply:
It has nice images of game boxes

Reply:
@einstein95 I wish they had a way to organize by engine. This resource was a huge help thank you